config/locales/client.en.yml

```diff
@@ -1,11 +0,0 @@
-en:
```

Comment:
I didn't remove it >>> may be by mistake I thought I already fork it  :) 

sorry if that happen